docs/cli-reference/dfx-nns.md

```diff
@@ -0,0 +1,119 @@
+
+Use the `dfx nns` subcommands to interact with the Network Nervous System.
+
+The basic syntax for running `dfx nns` commands is:
+
+``` bash
+dfx nns [subcommand] [flag]
+```
+
+Depending on the `dfx nns` subcommand you specify, additional arguments, options, and flags might apply. For reference information and examples that illustrate using `dfx nns` commands, select an appropriate command.
+
+| Command                             | Description                                                                   |
+|-------------------------------------|-------------------------------------------------------------------------------|
+| [`import`](#_dfx_nns_import)        | Adds the NNS canisters to the local dfx.json as remote canisters.             |
+| [`install`](#_dfx_nns_install)      | Deploys NNS canisters to the local dfx server                                 |
+| `help`                              | Displays usage information message for a specified subcommand.                |
+
+To view usage information for a specific subcommand, specify the subcommand and the `--help` flag. For example, to see usage information for `dfx nns install`, you can run the following command:
+
+``` bash
+$ dfx nns install --help
+```
+
+
+## dfx nns import
+
+Use the `dfx nns import` command to add the NNS canisters to the local `dfx.json`.  It also downloads the did files and sets the canister IDs of the NNS cansiters so that you can make API calls to NNS canisters.
+
+### Basic usage
+
+``` bash
+$ dfx nns import
+```
+
+### Flags
+
+You can use the following optional flags with the `dfx nns import` command.
+
+| Flag                | Description                                   |
+|---------------------|-----------------------------------------------|
+| `-h`, `--help`      | Displays usage information.                   |
+| `--network-mapping` | Renames networks when installing canister ids |
+| `-V`, `--version`   | Displays version information.                 |
```

Comment:
```suggestion
| `-h`, `--help`      | Displays usage information.                   |
| `--network-mapping` | Renames networks when installing canister IDs. |
| `-V`, `--version`   | Displays version information.                 |
```
Also, the `--network-mapping` flag is really confusing (at least for me). Could you add an example below that describes how this exactly behaves?

Reply:
Personally I feel as if the `--network-mapping` needs a bit of a review.  In general, I believe that it can be useful but right now there aren't any use cases for it and the exact behaviour in various cases isn't clear to me yet.  E.g. how do I import all networks listed remotely?  What happens if the target network doesn't exist?  I believe there maybe will be interesting use cases when end users have remote testnets.  At the moment I'd rather not document it or document it minimally until we have worked out how we really want it to work.  But I can document what I know and then we can decide how to proceed.